code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -739,7 +743,7 @@
 	description = "These restorative peptides not only speed up wound healing, but are nutrious as well!"
 	nutriment_factor = 10 * REAGENTS_METABOLISM // 33% less than nutriment to reduce weight gain
 	brute_heal = 3
-	burn_heal = 1
+	tox_heal = 1
```

Comment:
Why

Reply:
having trophazole gives you toxin, this heals the toxin